CONTRIBUTING.md

```diff
@@ -89,7 +89,7 @@ It’s possible that you found a bug or enhancement that we haven’t posted to
     ```
 9. [Open a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork), and make sure to fill in the required fields.
 10. An Expensify engineer and a member from the Contributor-Plus team will be assigned to your pull request automatically to review.
-11. Provide daily updates until reaching completion of your PR.
+11. Daily updates on weekdays are highly recommended. If you know you won’t be able to provide updates for > 1 week, please comment on the PR or issue how long you plan to be out so that we may plan accordingly. We understand everyone needs a little vacation here and there. Any issue that doesn't receive an update for 1 full week may be considered abandoned and the original contract terminated.
```

Comment:
I could be wrong, but I thought @marcaaron 's comment [here](https://github.com/Expensify/App/pull/6887#discussion_r774718599) was suggesting we replace these two sentences:

> If you know you won’t be able to provide updates for > 1 week, please comment on the PR or issue how long you plan to be out so that we may plan accordingly. We understand everyone needs a little vacation here and there.

With this 1 sentence:

> Any issue that doesn't receive an update for 1 full week may be considered abandoned and the original contract terminated.

That being said, I'm pretty happy with the full text we have now 👍

Reply:
His comment posted 3 new phrases compared to the earlier so I read it like so,
We can replace,
> Daily updates on weekdays are expected.

with
> Daily updates on weekdays are highly recommended.

Replace,
> If there is a plan for no updates for a while we expect the courtesy to be informed about the duration there will be no work so that we may plan accordingly.

with,
> If you know you won’t be able to provide updates for > 1 week, please comment on the PR or issue how long you plan to be out so that we may plan accordingly. We understand everyone needs a little vacation here and there.

Replace,
> Any issue without an update for 1 week with no explanation why there is no progress will be considered abandoned and will likely be closed or reassigned, and the original contract will be terminated.

with,
> Any issue that doesn't receive an update after 1 week may be considered abandoned and the original contract terminated.

So thats how i inferred it. We can wait for @marcaaron to confirm as well too but if we update it such the point is only,
> Any issue that doesn't receive an update after 1 week may be considered abandoned and the original contract terminated.

then i feel like it doesn't capture this point which i feel is important,
> If you know you won’t be able to provide updates for > 1 week, please comment on the PR or issue how long you plan to be out so that we may plan accordingly. We understand everyone needs a little vacation here and there.

Reply:
I like where this landed. Nice edits @chiragsalian!